server/main.go

```diff
@@ -30,6 +32,9 @@ var (
 	port                  = flag.String("port", ":8080", "port to listen on")
 	databasePath          = flag.String("db", "./db/development.db", "path to sqlite3 database file")
 	templatesPath         = flag.String("templates", "./assets/templates", "path to template files")
+	env                   = flag.String("env", "production", "environment applikatoni is used in")
```

Comment:
I think `development` is better as default. Much higher rate of starting/stopping in development than in production.